client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,15 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <NavLink to={`/movies/${movie.id}`}>
+            <span key={movie.id} className="saved-movie">
```

Comment:
we should member that our key should be on the first element after using map method